credits.txt

```diff
@@ -9,8 +9,26 @@ https://endless-sky.github.io
 The game's Steam forums
 Discord (invite: ZeuASSx)
 
-Programming
+Creator
   Michael Zahniser
+  
+Developers
+  Amazinite
+  Pointedstick
+  tehhowch
+
+Programming
```

Comment:
<details><summary>Here's everyone who have made any changes to the /source/ files. I think we should decide what we want to consider "Programming" though, as I wouldn't necessarily call all changes to the source files programming contributions (e.g. typo fixes in comments). There are also changes not made to the source files that should probably be considered programming (e.g. updating the xCode files).</summary>

abenkovskii
AdamKauffman
akien-mga
Amazinite
AMDmi3
bojidar-bg
ChamEV
colbygallup
comnom
CyberShadow
db47h
Disiuze
dplepage
DrBlight
dreness
dzhu
Elyssaen
EndrosG
fcfort
flaviojs
Fzzr
guillochon
Hacklin
Hadron1776
ItsNickBarry
Janaszar
jmhorjus
jostephd
jozef-mitro
kozbot
lheckemann
MageKing17
MarcelineVQ
mattsoulanille
MCOfficer
Nescio0
NomadicVolcano
oo13
OverYrPaygrade
petervdmeer
pscamman
Rakete1111
reizbar
rlane
rzahniser
sdennie
sfiera
tboby
thomasballinger
TurkeyMcMac
willbeason
zwparchman
</details>

Reply:
Agreed. That being said, exactly what counts as "programming" I think is probably up to you Dev's; and tehhowch in particular.

As far as that goes, I think inclusion in specific sections like "Programming" or "Writers" or "Editors" should be something that we actively manage. So someone doesn't get put in "Writers" as soon as they've created one job. The category would be for people who have made (or are) making a continued and/or long-lasting impact on the story. Likewise for Programmers. If I make one small change, maybe add a single attribute somewhere, I don't think I'd merit inclusion on the list of programmers. If I improve my coding skills and start making regular contributions that are deemed beneficial/useful, though... 

Basically, what I'm getting at is that each of these categories should be places where we are highlighting people that have made a significant contribution to that area of the game.

Reply:
Well if we're going to curate these lists to just major contributions, then I'd say the current writers list could drop some names, as I only suggested all those who had simply created missions instead of all those who had contributed something I'd consider significant (as here I've simply listed all those who've made changes to the source code instead of only listing significant contributors): Czartrak and MasterOfGrey only added a few jobs; MessyMix, Anarchist2, and Fzzr have only added one mission each; and Jana's contribution to the Nanachi missions could more appropriately be considered to fall under "Proof Readers & Editors".

Reply:
OK, I removed those and moved Jana over to Editors.

I don't really know a whole lot about how things work on the programming side; if you thing it'd make more sense to have separate sections for new mechanics and major bug fixes, I'm fine with that. Just let me know who should go on each list.

Reply:
<details><summary>Here's a shortened list from the one above, with reasons for each.</summary>

AdamKauffman (#4472 was sizable.)
colbygallup (#772 You can thank him for piercing weapons.)
comnom (Various fixes/changes.)
CyberShadow (#3727 made zooming better and I like that.)
db47h (Various fixes/changes.)
Elyssaen (Various fixes/changes.)
EndrosG (" ")
flaviojs (" ")
guillochon (" ")
Hacklin (#2982 is responsible for how movement works right now. Might not be enough for others, but given #4924, I realize #2982 has actually been fairly impactful.)
Hadron1776 (Various fixes/changes,)
ItsNickBarry (#922 #923 responsible for solar collection and solar panel outfits.)
kozbot (Various fixes/changes.)
MarcelineVQ (" ")
NomadicVolcano (" ")
oo13 (#4538)
petervdmeer (#4370 ongoing auto testing work.)
tehhowch (Self-explanatory.)
thomasballinger (#3828 improved zoom further. Also, wholesome tweet.)

When it came to "various fixes/changes," I basically looked for if they'd contributed more than just one or two minor changes, so if I gave that as the reasoning then that person has contributed at least a handful of changes big and small. We can trim this list down further too if we want; I'd perhaps drop the names of those where my reasoning is that they contributed a single, small mechanic/change, e.g. colbygallup, Hacklin, ItsNickBarry, thomasballinger. I've already omitted pscamman, who I considered for having done #3802, but that's only a single two line change, so I don't think it met the bar.
</details>

Reply:
OK. I have renamed the category "Major Programming" and replaced the previous list with this new one, with some names removed (the ones you listed there as being small changes)

Also renamed the "Missions & Story Creators" to "Story Creators" because I think that's a bit more clear that these aren't just the people who have made a single mission or two; they're actually writing stories.